engine/runtime-compiler/src/main/java/module-info.java

```diff
@@ -0,0 +1,17 @@
+module org.enso.runtime.compiler {
+  requires java.logging;
+  requires scala.library;
+
+  requires org.enso.engine.common;
+  requires org.enso.pkg;
+  requires org.enso.runtime.parser;
+
+  requires org.openide.util.lookup.RELEASE180;
```

Comment:
Dependencies on `org.openide.util.lookup` are probably just `static` in all cases except `persistance-api`.